### PR TITLE
CI enhancements

### PR DESCRIPTION
**Pull request by shyamjesal**
added commitlint action to check for signoffs.
fixed gofmt issues.
removed calico pod issue from quickstart guide.
Swap turned off in cri runner container as it is not supported by kind clusters

Signed-off-by: Shyam Jesal <s.jesalpura@gmail.com>

### CODE REVIEW

File: .github/workflows/lint_metadata.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@master
+    - uses: rojopolis/spellcheck-github-actions@0.5.0
+      name: Spellcheck
+      with:
+        config_path: configs/.spellcheck.yml
+  commitlint:
+    name: Commitlint
+    runs-on: ubuntu-latest
```
Comment (ustiugov) [inline .github/workflows/lint_metadata.yml:20]: please use ubuntu18 as elsewhere

File: .github/workflows/lint_metadata.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+name: Spellcheck + Commitlint
```
Comment (ustiugov) [inline .github/workflows/lint_metadata.yml:1]: let's call all of these linters.yml and "Linters" name.

File: configs/commitlint.config.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+const Configuration = {
+    /*
+     * Any rules defined here will override rules from @commitlint/config-conventional
+     */
+    rules: {
+        'signed-off-by': [1, 'always', 'Signed-off-by:'],
+    },
+};
```
Comment (ustiugov) [inline configs/commitlint.config.js:8]: can you please add ```header-max-length``` of 72 and ```header-min-length``` of 10

File: scripts/github_runner/setup_cri_dev_env.sh [hunk 4]
```diff
@@ -46,6 +46,7 @@ net.bridge.bridge-nf-call-ip6tables = 1
 EOF
 
 sudo sysctl --system
+sudo swapoff -a
```
Comment (ustiugov) [inline scripts/github_runner/setup_cri_dev_env.sh:49]: do mention this in the PR description

**Review by ustiugov — CHANGES_REQUESTED** [review]

File: .github/workflows/lint_metadata.yml (hunk 1 above)
Comment (shyamjesal) [reply .github/workflows/lint_metadata.yml:20]: done

File: .github/workflows/lint_metadata.yml (hunk 2 above)
Comment (shyamjesal) [reply .github/workflows/lint_metadata.yml:1]: done

File: configs/commitlint.config.js (hunk 3 above)
Comment (shyamjesal) [reply configs/commitlint.config.js:8]: added

**Review by ustiugov — APPROVED** [review]

**Pull request merged**